Makefile

```diff
@@ -16,7 +16,7 @@
 #
 
 # The target to build, see VALID_TARGETS below
-TARGET    ?= REVO
+TARGET    ?= OMNIBUSF4V3
```

Comment:
Why change default target?

Reply:
This can be dicarded, of course. It's was just for my purposes.